packages/core/admin/admin/src/utils/tests/getFetchClient.test.js

```diff
@@ -0,0 +1,27 @@
+import { auth } from '@strapi/helper-plugin';
+import { getFetchClient } from '../getFetchClient';
+
+const token = 'coolToken';
+auth.getToken = jest.fn().mockReturnValue(token);
```

Comment:
Is this necessary now?

Reply:
yes, it is needed, otherwise a receive in the test `Bearer undefined`

Reply:
I have removed the mock of the auth.clearAppStorage